setup-mini.cfg

```
@@ -103,7 +103,7 @@ install_requires =
     pymaven_patch >= 0.2.8
     requests >= 2.7.0
     saneyaml >= 0.5.2
-    spdx_tools == 0.7.0a3
+    spdx_tools >= 0.7.0rc0, ==0.7.*
```

We avoid using upper cap in setup requirements. We pin instead in requirements files.
```suggestion
    spdx_tools >= 0.7.0rc0
```
See https://iscinumpy.dev/post/bound-version-constraints/ and https://caremad.io/posts/2013/07/setup-vs-requirement/ for detailed articles on the topics. ScanCode is both used as an app and as a library. App deps are pinned, library deps are not capped.

---

Since there is currently a big refactoring in the progress, which is  breaking the API after 0.7, we think that it is safer to restrict it here. Otherwise we will run into the same race condition with releasing.

Pinning in requirement files was not sufficient to mitigate the breaking changes between `0.7.0a3` and `0.7.0rc0`.